src/pages/ChatListPage.jsx

```diff
@@ -85,23 +86,22 @@ export default function ChatListPage() {
     );
   }
   return (
-    <div className="flex flex-col p-7 w-full h-full">
-      <h1 className=" font-bold text-xl pb-5">채팅</h1>
+    <div className="flex flex-col w-full h-full">
+      <ChatListHeaderRow />
       {/* 채팅 목록 영역 */}
-      <div className=" overflow-y-auto mb-16">
-        {chatList.length > 0 &&
-          chatList?.map((message) => (
-            <div key={message.timestamp}>
-              <ChatRoomItem
-                timestamp={message.timestamp}
-                counterName={message.counterName}
-                lastMessage={message.lastMessage}
-                chatId={message.chatId}
-                unreadCount={message.unreadCount}
-              />
-            </div>
-          ))}
-      </div>
+      {chatList.length > 0 &&
+        chatList?.map((message) => (
```

Comment:
length 체크를 굳이 하지 않아도 될 것 같습니다! optional chainning으로 nullable한 케이스는 대응이 되었고
map 메소드도 Length가 0인 경우에 루프를 돌지 않기 때문입니다

```suggestion
      {chatList?.map((message) => (
```